### ISSUE TITLE
how to get the dataset

### ISSUE THREAD

**Issue by magicmarkov** [AyanKumarBhunia/Sketch2Saliency#2]
![屏幕截图 2024-10-09 191441](https://github.com/user-attachments/assets/a6c7b185-3c9e-4868-b471-c8327cdac546)
Excuse me, how can I get this(sketchy_all.pickle) , use SketchyDatabase?

**Comment by AyanKumarBhunia** [comment]
Yes... This is just sketchy dataset preprocessed in pickle format.

**Comment by magicmarkov** [comment]
Ok, thank you!
